src/saga/adaptors/pbspro/pbsprojob.py

```diff
@@ -826,7 +831,7 @@ def _job_get_info(self, job_id, reconnect):
 
         # run the PBS 'qstat' command to get some infos about our job
         # TODO: create a PBSPRO/TORQUE flag once
-        if 'PBSPro_1' in self._commands['qstat']['version']:
+        if 'PBSPro_1' in self._commands['qstat']['version'] or '14.2.4' in self._commands['qstat']['version']:
```

Comment:
Would it make sense to check for `14.2` so that it does not break on the next subminor update?

Reply:
Yea, I agree. Making the change and sending PR to master.